src/gsy_e/models/strategy/future/strategy.py

```diff
@@ -263,10 +265,10 @@ def event_tick(self, strategy: "BaseStrategy") -> None:
 
 def future_market_strategy_factory(
         asset_type: AssetType,
-        initial_buying_rate: float = FutureTemplateStrategiesConstants.INITIAL_BUYING_RATE,
-        final_buying_rate: float = FutureTemplateStrategiesConstants.FINAL_BUYING_RATE,
-        initial_selling_rate: float = FutureTemplateStrategiesConstants.INITIAL_SELLING_RATE,
-        final_selling_rate: float = FutureTemplateStrategiesConstants.FINAL_SELLING_RATE
+        initial_buying_rate: float = None,
+        final_buying_rate: float = None,
+        initial_selling_rate: float = None,
+        final_selling_rate: float = None
```

Comment:
NIT:
```suggestion
        initial_buying_rate: Optional[float] = None,
        final_buying_rate: Optional[float] = None,
        initial_selling_rate: Optional[float] = None,
        final_selling_rate: Optional[float] = None
```

Reply:
Right! Thank you, I will keep this in mind. Although I've just removed these arguments completely after this disscussion https://github.com/gridsingularity/gsy-e/pull/1450#discussion_r897735593.

Reply:
Ah, ok. I musst have looked at a wrong commit again. Sorry. I think the goal behind having these as construction parameters, in oder to be prepared for the future, when the user wants to adapt these per area and not globally. This is why the former version made more sense to me. But as this is just a minor change, we can alway change it back later.